src/framework/webserver/core.clj

```diff
@@ -0,0 +1,52 @@
+(ns framework.webserver.core
+  (:require
+    [framework.config.core :as config]
+    [framework.interceptor.queue :as interceptor.queue]
+    [framework.route.core :as route]
+    [framework.state.core :as state]
+    [ring.adapter.jetty :as jetty]
+    [xiana.core :as xiana]))
+
+;; web server reference
+(defonce -webserver (atom {}))
+
+
+(defn handler-fn
+  "Return jetty server handler function."
+  [deps]
+  (fn [http-request]
+    (let [state (state/make deps http-request)
+          queue (list #(route/match %)
+                      #(interceptor.queue/execute % (:controller-interceptors deps)))]
+      (-> (xiana/apply-flow-> state queue)
+          ;; extract
+          (xiana/extract)
+          ;; get the response
+          (get :response)))))
```

Comment:
make the handler function compatible with async ring, I edited this in the PR so make sure it compiles.

```suggestion
(defn handler-fn
  "Return jetty server handler function."
  [deps]
  (fn handle*
   ([http-request]
    (let [state (state/make deps http-request)
          queue (list #(route/match %)
                      #(interceptor.queue/execute % (:controller-interceptors deps)))]
      (-> (xiana/apply-flow-> state queue)
          ;; extract
          (xiana/extract)
          ;; get the response
          (get :response))))
    ([request respond _]
      (respond (handle* request)))))
```